packages/@aws-cdk/aws-events-targets/lib/api-destination.ts

```diff
@@ -0,0 +1,66 @@
+import * as events from '@aws-cdk/aws-events';
+
+/**
+ * Customize the EventBridge Api Destinations Target
+ * @experimental
+ */
+export interface ApiDestinationProps {
+  /**
+   * The event to send
+   *
+   * @default the entire EventBridge event
+   */
+  readonly event?: events.RuleTargetInput;
+
+  /**
+   * The headers that need to be sent as part of request invoking the API Gateway REST API
+   * or EventBridge Api destinations.
+   *
+   * @default none
+   */
+  readonly headerParameters?: { [key: string]: string };
+  /**
+   * The path parameter values to be used to populate API Gateway REST API
+   * or EventBridge Api destinations path wildcards ("*")
+   *
+   * @default none
+   */
+  readonly pathParameterValues?: string[]
+  /**
+   * The query string keys/values that need to be sent as part of request invoking the API Gateway REST API
+   *  or EventBridge Api destination.
+   *
+   * @default none
+   */
+  readonly queryStringParameters?: { [key: string]: string };
+}
+
+/**
+ * Use an API Destination rule target.
+ * @experimental
+ */
+export class ApiDestination implements events.IRuleTarget {
```

Comment:
I would suggest to rename this to EventBridgeApiDestination and pass ApiDestination as parameter, same as the pattern of CloudWatchLogGroup and LogGroup.